src/components/ArticleDetails.jsx

```diff
@@ -19,13 +20,21 @@ const ArticleDetails = () => {
 				setIsError(true);
 				setIsLoading(false);
 			});
+
+		getComments(article_id)
```

Comment:
Suggestion: we could have a separate loading state for the comments, or you could wait to set loading to false when both the articles and comments have loaded?